dependency_support/org_theopenroadproject_asap7/asap7sc6t_rev26.bzl

```diff
@@ -0,0 +1,264 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" ASAP7 "rev 26" 6 track standard cell library (with SRAM blocks). """
+
+# Layouts for GDS generation
+# ------------------------------------------------------------------------
+
+# Standard cells
+filegroup(
+    name = "gds_cells_rvt",
+    srcs = ["asap7sc6t_26/GDS/asap7sc6t_26_R.gds"],
+    visibility = [":data_visibility"],
+)
+
+filegroup(
+    name = "gds_cells_lvt",
+    srcs = ["asap7sc6t_26/GDS/asap7sc6t_26_L.gds"],
+    visibility = [":data_visibility"],
+)
+
+filegroup(
+    name = "gds_cells_slvt",
+    srcs = ["asap7sc6t_26/GDS/asap7sc6t_26_SL.gds"],
+    visibility = [":data_visibility"],
+)
+
+# SRAM
+filegroup(
+    name = "gds_sram",
+    srcs = ["asap7sc6t_26/GDS/asap7sc6t_26_SRAM.gds"],
+    visibility = [":data_visibility"],
+)
+
+# Timing information (in compressed Liberty format) for synthesis and static
+# timing analysis (STA).
+# ------------------------------------------------------------------------
+
+# Standard cells
+filegroup(
+    name = "libgz_cells_rvt",
+    srcs = glob(["asap7sc6t_26/LIB/CCS/*RVT*.lib.gz"]),
+    visibility = [":data_visibility"],
+)
+
+filegroup(
+    name = "libgz_cells_lvt",
+    srcs = glob(["asap7sc6t_26/LIB/CCS/*LVT*.lib.gz"]),
+    visibility = [":data_visibility"],
+)
+
+filegroup(
+    name = "libgz_cells_slvt",
+    srcs = glob(["asap7sc6t_26/LIB/CCS/*SLVT*.lib.gz"]),
+    visibility = [":data_visibility"],
+)
+
+# SRAM
+filegroup(
+    name = "libgz_sram",
+    srcs = glob(["asap7sc6t_26/LIB/CCS/*SRAM*.lib.gz"]),
+    visibility = [":data_visibility"],
+)
+
+# FIXME: What about NLDM liberty?
```

Comment:
Prefer using NLDM over CCS for OpenROAD. We don't have support for CCS yet so it's not adding a lot of value, but does consume a good bit of disk.